zip-tests/tests.sh

```diff
@@ -0,0 +1,393 @@
+#!/bin/bash
```

Comment:
We should add bash as a dependency in buildenv if bash is really needed or change it to sh if not

Reply:
Also, maybe put the source of where the original tests.sh was taken from in case we want to update at some point

Reply:
I have added the repo details in the script and also removed bash and used sh for the script.